.vscode/launch.json

```diff
@@ -16,7 +16,10 @@
             "outFiles": [
                 "${workspaceFolder}/out/**/*.js"
             ],
-            "preLaunchTask": "npm: watch"
+            "preLaunchTask": "npm: watch",
+            "skipFiles": [
```

Comment:
Is this added intentionally?  If so, what's it for?

Reply:
When debugging the extension, and stepping into functions (F11) it skippes internal node code. It makes debugging easier